doc/nrf/libraries/dfu/suit_dfu.rst

```diff
@@ -0,0 +1,54 @@
+.. _subsys_suit:
```

Comment:
Is an application expected to use this library directly or through a helper like dfu_target?

Reply:
Directly. DFU target uses this API to implement it's functionality.

Reply:
Now I'm confused. What is the intended call flow?

* application -> suit_dfu
* application -> dfu_target -> suit_dfu
* something else?

Reply:
The first and the second option are a valid use cases

Reply:
Which one is the recommended one?

Reply:
OK, I've added a note about the dfu_target compatibility layer.
In fact, it does not use the SUIT DFU library and allows only for a single-push updates.